examples/LanguageModel/sketch.js

```diff
@@ -0,0 +1,61 @@
+let lm;
+let curWord = 0;
+let angle = 0;
+let radius = 60;
+
+function setup() {
+  createCanvas(400, 400);
+  background(0);
+
+  lm = ml5.languageModel(onModelLoaded);
```

Comment:
Agreed it would be great to support `preload()`. 

Also, I wonder if we should require a "string" that references a specific model to load. Even though this is perhaps an extra, unecessary step it emphasizes to the user that this isn't magic, and requires an acknowledgement of the specific model they are using. Some options:

```javascript
  lm = ml5.languageModel("TinyStories", onModelLoaded);
  lm = ml5.languageModel("TinyLlamas", onModelLoaded);
```

Referencing the model name is probably more important, but it's also important to emphasize the specific dataset that was used for training.